aldryn_style/models.py

```diff
@@ -81,41 +101,48 @@ def __unicode__(self):
             display = u'{0} ({1})'.format(display, self.additional_class_names)
         if self.label:
             display = u'“{0}”: {1}'.format(self.label, display)
-        return u"%s" % display
+        return u'%s' % display
 
     def inline_style(self):
-        style = ""
+        style = ''
         if self.padding_left:
-            style += "padding-left: %dpx; " % self.padding_left
+            style += 'padding-left: %dpx; '.format(self.padding_left)
```

Comment:
won't work, since format fills ``'{0}'``, ``'{name}'`` or other formatters, not ``%d``. If you want to specify digit only please use ``'{0:d}'``.